code/_onclick/hud/rendering/render_plate.dm

```diff
@@ -374,6 +376,23 @@
 	plane = RENDER_PLANE_NON_GAME
 	render_relay_planes = list(RENDER_PLANE_MASTER)
 
+/atom/movable/screen/plane_master/rendering_plate/emissive_bloom
+	name = "Emissive bloom plate"
+	documentation = "Plate used to bloom emissives before adding them onto the overlay lighting plane. This relies on game rendering plate,\
+		 which assembles all in-world planes affected by lighting, is above the emissive plane - which allows us to bypass having to fullbright\
+		 the emissive plane on a separate plate before alpha masking the rendering plate copy via multiplying fullbright emissive created here\
+		 with the game plate, which technically achieves the same effect. Odd, but it works, and is significantly graphically cheaper."
```

Comment:
requires `<br>`s on linebreaks for the actual ui.

Reply:
This doesn't need linebreaks on the UI, these are not separate paragraphs. It renders just fine.